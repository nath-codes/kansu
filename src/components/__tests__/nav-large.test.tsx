import 'jest-styled-components'
import React from 'react'

import NavLarge from '../nav-large'
import { renderWithTheme } from './helpers'

describe('NavLarge', () => {
  it('renders correctly', () => {
    const tree = renderWithTheme(<NavLarge />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
