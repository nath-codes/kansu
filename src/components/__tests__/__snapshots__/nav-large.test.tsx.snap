// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavLarge renders correctly 1`] = `
.c3 {
  color: #49555e;
  display: block;
  font-family: "Roboto",sans-serif;
  font-weight: bold;
  font-size: 1rem;
  -webkit-letter-spacing: 2.3px;
  -moz-letter-spacing: 2.3px;
  -ms-letter-spacing: 2.3px;
  letter-spacing: 2.3px;
  position: relative;
  text-transform: uppercase;
  -webkit-transition: color 0.3s;
  transition: color 0.3s;
}

.c3:hover {
  color: #ffffff;
}

.c3.active {
  color: #ffffff;
  position: relative;
}

.c3.active:after {
  border-bottom: solid 0.375rem #ffffff;
  bottom: -0.375rem;
  content: '';
  left: 0;
  position: absolute;
  right: 0;
}

.c0 {
  display: none;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c2 {
  margin-right: 1.5rem;
}

.c2:last-child {
  margin-right: 0;
}

@media only screen and (min-width:980px) {
  .c3.active:after {
    bottom: -0.75rem;
  }
}

@media only screen and (min-width:768px) {
  .c0 {
    display: block;
    font-family: "Roboto",sans-serif;
  }
}

<nav
  className="c0"
>
  <ul
    className="c1"
  >
    <li
      className="c2"
    >
      <a
        activeClassName="active"
        aria-label="Home page"
        className="c3"
        href="/"
      >
        Home
      </a>
    </li>
    <li
      className="c2"
    >
      <a
        activeClassName="active"
        aria-label="About page"
        className="c3"
        href="/about"
      >
        About
      </a>
    </li>
    <li
      className="c2"
    >
      <a
        activeClassName="active"
        aria-label="Clients page"
        className="c3"
        href="/clients"
      >
        Clients
      </a>
    </li>
    <li
      className="c2"
    >
      <a
        activeClassName="active"
        aria-label="Contact page"
        className="c3"
        href="/contact"
      >
        Contact
      </a>
    </li>
  </ul>
</nav>
`;
